content/docs/04-clusters/1-new-clusters/04-aks.md

```diff
@@ -73,61 +83,141 @@ To create an Azure cloud account, we need:
+    | **Resource Group** | Select the resource group in which the cluster should be deployed.                           |
+    | **SSH Key**        | Public key to configure remote SSH access to the nodes.                                      |
+    | **Placement**      | You may leave this unchecked, unless the choice of placement is Static, then select:<br /> **Virtual Network**: Select the virtual network from dropdown menu. <br /> **CIDR Block**: Enter the groups of addresses: <br /> **Control plane Subnet**: Select the control plane network from the dropdown menu. <br /> **Worker Network**: Select the worker network from the dropdown menu.  <br /> **Update worker pools in parallel**: Check the box to concurrently update the worker pools.|
+    
+
+7.  Configure the worker node. 
+
+
+<br />
 
-The following steps need to be performed to provision a new AKS cluster:
+## Adding a Worker Node
 
-1. Provide the basic cluster information like Name, Description, and Tags.
+1. Configure the worker node. In this section, we will learn how to configure a worker node. However, in a production environment, the ideal settings are to create a node pool with at least three (3) nodes. In that case, a System Node Pool must be created first.
+
+<InfoBox>
+The System Node Pool serves to run critical system components. Its operating system type must be created prior to other worker node pools. Palette automatically determines this when you turn it into a system node.
+</InfoBox>
+
+2. Click the checkbox to turn this into the System Node if you are creating a node pool with multiple worker nodes.
+
+    **Note**: Identifying the System Node Pool as such will deactivate the **Cloud Configuration** section and disable the ability to select the OS. Notice the Enable Autoscaler is disabled, as well.
+
+
+3. Provide a name in the **Node pool name** text box. When creating a system node, it is a good idea to include an identifying name as such.
+
+
+4. Add the **Desired size**. You can start with three for multiple nodes.
+
+
+5. Include **Additional Labels**. This is optional.
+
+
+6. In the **Azure Cloud Configuration** section, add the **Instance type**. The cost details present for a review.
+
+
+7. Enter the **Managed Disk information** and its size.
+
+
+8. If you are including additional or multiple nodes to make a node pool, then click the **Add Worker Nodes** button to create the next node.
 
 
-2. Select a Cluster Profile created for the AKS Cluster. The profile definition will be used as the Cluster Construction Template.
+## Include Additional Nodes to Create a Node Pool
 
+1. Identify the next node pool as a worker node and give it a worker name. 
 
-3. Review and override pack parameters as desired. By default, parameters for all packs are set with values defined in the Cluster Profile.
 
-4. Provide the Azure Cloud account and placement information.
+2. This time, you are able to enable **Autoscaler** and System Node Pool will not be enabled.

```

Comment:
" This time, you are able to enable **Autoscaler**"
 autoscaler is available for system node pool as well